models/cm_default.h

```diff
@@ -146,10 +155,11 @@ The following parameters can be set in the status dictionary.
 The following parameters can be used when adding compartments using ``SetStatus()``
 
 =========== ======= ===============================================================
- C_m        uF      Capacitance of compartment (default: 1 uF)
+ C_m        nF      Capacitance of compartment (default: 1 nF)
  g_C        uS      Coupling conductance with parent compartment (default: 0.01 uS)
  g_L        uS      Leak conductance of the compartment (default: 0.1 uS)
  e_L        mV      Leak reversal of the compartment (default: -70. mV)
+ V_init     mV      Initialization voltage of the compartment (default: -75. mV)
```

Comment:
Why is the default different from the leak potential? Doesn't this cause a transient at the start of the simulation?

Reply:
I've set the default to -70 mV now. This would've indeed caused a transient which now won't happen if there are no other conductances. Nevertheless, when there are non-zero ion channels the equilibrium will most likely not be equal to the initialization, nor the leak reversal, so there will be a transient anyway.